### ISSUE TITLE
JIRA Issue: FOQUS and TurbineUpload windows blank/not responding while in SinterConfig GUI

### ISSUE THREAD

**Issue by fsoepyan** [CCSI-Toolset/FOQUS#38]
From JIRA:
- Reporter: Natalia Kitch
- Assignee: Megha Sandesh
- Original title: FOQUS and TurbineUpload windows blank/not responding while in SinterConfig GUI
- Created: 16/Oct/14 4:22 PM
- Updated: 13/Jan/16 8:45 AM

Description:

FOQUS.Session->Add/Update Turbin Model->create/edit SinterConfig File ->Sinter Config GUI->browse->select file->loading...
While the above is in process:
1) FOQUS and TurbineUpload windows are blanked out;
2) clicking on them -> still blank with "not responding" window banners

Attachments

<img width="955" alt="foqus-simsinter-turbine" src="https://user-images.githubusercontent.com/30811140/31097647-fa2a184a-a78d-11e7-83a9-9a7397800eb0.PNG">

Chat history:

Natalia Kitch (Inactive) added a comment - 16/Oct/14 4:28 PM 

NK: added a snapshot.

John Eslick added a comment - 17/Oct/14 8:02 AM 

I think that is normal. It could probably be done better, but the upload window is waiting on the sinter config gui to return. I'll try to fix this in the next version.

**Comment by ksbeattie** [comment]
Closing this as something we won't fix.
